src/umpire/strategy/DynamicPool.hpp

```diff
@@ -50,32 +60,50 @@ class DynamicPool :
      * \param min_initial_alloc_size The minimum size of the first allocation
      *                               the pool will make.
      * \param min_alloc_size The minimum size of all future allocations.
+     * \param h_fun Heuristic callback function.
```

Comment:
h_fun is probably an unnecessarily short argument name, can it be slightly more descriptive?